ci-operator/step-registry/openshift/image-based/upgrade/target/openshift-image-based-upgrade-target-ref.yaml

```diff
@@ -22,10 +22,10 @@ ref:
     default: quay.io/openshift-kni/image-based-seeds
     documentation: The repo where the seed image will be uploaded. The backup secret needs to have access to this repo
   dependencies:
-  - name: lca-operator
-    env: LCA_PULL_REF
+  - name: lifecycle-agent-operator-bundle
+    env: OO_BUNDLE
```

Comment:
Why `OO_BUNDLE`?

Reply:
I just thought that it's better to follow the naming convention found in the respective [docs](https://docs.ci.openshift.org/docs/how-tos/testing-operator-sdk-operators/).